src/Sarif.TestUtilities/FileDiffingTests.cs

```diff
@@ -27,9 +27,9 @@ public static string GetTestDirectory(string subdirectory = "")
 
         // Retrieving the source path of the tests is only used in developer ad hoc
         // rebaselining scenarios. i.e., this path won't be consumed by AppVeyor.
-        public static string GetProductTestDataDirectory(string subdirectory = "")
+        public static string GetProductTestDataDirectory(string testBinaryName, string subdirectory = "")
         {
-            return Path.GetFullPath(Path.Combine(@"..\..\..\..\..\src\Sarif.UnitTests\TestData", subdirectory));
+            return Path.GetFullPath(Path.Combine($@"..\..\..\..\..\src\{testBinaryName}\TestData", subdirectory));
```

Comment:
>testBinaryName [](start = 72, length = 14)

baseline file location was previously always getting redirected to sarif.unit tests location.

Reply:
Nice generalization.

---
In reply to: [249130208](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249130208) [](ancestors = 249130208)